dwarf/reader/reader.go

```diff
@@ -144,3 +144,28 @@ func (reader *Reader) NextMemberVariable() (*dwarf.Entry, error) {
 	// No more items
 	return nil, nil
 }
+
+// NextPackage moves the reader to the next debug entry that describes a package variable
```

Comment:
Change to `NextPackageVariable`, and add a period at the end (nit-picky I know, sorry).